src/PrettierEditProvider.ts

```diff
@@ -35,21 +35,24 @@ async function hasPrettierConfig(filePath: string) {
     return config !== null;
 }
 
-type ResolveConfigResult = { config: PrettierConfig | null, error?: Error };
+type ResolveConfigResult = { config: PrettierConfig | null; error?: Error };
```

Comment:
Is this change correct? If so, than I should do some more research on typescript.

Reply:
Prettier did it.
interface or object type can separate their properties with `,` `;` or `\n`

Reply:
Interesting!